mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -22,6 +22,12 @@ CLASS net/minecraft/class_127 net/minecraft/entity/LivingEntity
 	METHOD method_917 getItemTexturePosition (Lnet/minecraft/class_31;)I
 	METHOD method_918 playAmbientSound ()V
 	METHOD method_919 onSpawnedFromSpawner ()V
+	METHOD method_926 getRotationVector (F)Lnet/minecraft/class_26;
+		ARG 1 delta
+	METHOD method_929 raycast (DF)Lnet/minecraft/class_27;
+		ARG 1 distance
+	METHOD method_931 getPositionVector (F)Lnet/minecraft/class_26;
```

Comment:
should these be getRotation and getPos or does that conflict/confuse it with other methods